src/pkg/artifactrash/dao/dao.go

```diff
@@ -16,9 +16,9 @@ type DAO interface {
 	// Delete the artifact trash specified by ID
 	Delete(ctx context.Context, id int64) (err error)
 	// Filter lists the artifact that needs to be cleaned
-	Filter(ctx context.Context) (arts []model.ArtifactTrash, err error)
+	Filter(ctx context.Context, timeWindow time.Time) (arts []model.ArtifactTrash, err error)
```

Comment:
I think the var name is misleading, timewindow means a range of time.

Reply:
updated.